data/campaigns/fri04.wmf/scripting/mission_thread.lua

```diff
@@ -87,6 +107,11 @@ function mission_thread()
    end
    sleep(2000)
    campaign_message_box(intro_3)
+   run(hurry_up)
```

Comment:
this should be moved down after the instantiation of the Objective to find reebaud as this is given in the intro_3 message